server/authentication/govukOneLogin.ts

```diff
@@ -19,6 +19,7 @@ passport.deserializeUser((user, done) => {
 const authenticationMiddleware = (): RequestHandler => {
   return async (req, res, next) => {
     if (req.isAuthenticated()) {
+      res.locals.user = req.user
```

Comment:
Unrelated change - just moving code to populate `res.locals.user` to make sure it happens if the user has actually been authenticated. Basically a tidy up after moving authentication middleware in #89 to support some routes not requiring authentication.